src/server/api/errors.js

```diff
@@ -30,6 +30,21 @@ export async function accessRequired(
   }
 }
 
+export async function assignmentOrAdminRequired(
+  user,
+  orgId,
+  assignmentId,
+  assignment
+) {
+  authRequired(user);
+  const isAdmin = await cacheableData.user.userHasRole(user, orgId, "ADMIN");
+  if (isAdmin || user.is_superAdmin) {
```

Comment:
In their review for #1297, @schuyler1d commented:

> schuyler1d 3 days ago  Collaborator
> this needs to be user.is_superadmin (it's a db field, and isn't case-changed)
> 
> Alternatively, you can delete it and just test on isAdmin, since the function below also tests superadmin.